TileTowns/Assets/Scenes/GameScene.unity

```diff
@@ -742,6 +742,65 @@ RectTransform:
   m_AnchoredPosition: {x: 0, y: 0}
   m_SizeDelta: {x: 0, y: 0}
   m_Pivot: {x: 0, y: 0}
+--- !u!1 &226715574
```

Comment:
i wont bother checking how its in a scene but i hope you just made a new object called RoadManger or something along side other managers added a script to it ;3